cmd/fleetctl/convert.go

```diff
@@ -61,22 +102,38 @@ func specGroupFromPack(name string, inputPack fleet.PermissivePackContent) (*spe
 			interval = uint(i)
 		}
 
+		// handle nil query.Platform
+		var queryPlatforms string
+		if query.Platform != nil {
+			queryPlatforms = *query.Platform
+		} else {
+			queryPlatforms = ""
+		}
```

Comment:
You don't need the `else` part, `var queryPlatforms string` declares the variable and it is always initialized with its "zero value", and in the case of a string that's the empty string.